### DISCUSSION TITLE
Add a feature to select frame number range for suggested frames

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1434] [Ideas]
Hi,

I occasionally have very long (1hr + ) videos in my project where I want to add random frames from a small time window. Most of the video the animal isn't doing much, but there's a specific 2 minute phase of the experiment where things are happening that the model has low-scoring skeletons for. 

In the Labelling Suggestions "Target" box, when you select "Current Video," can we have a submenu where we can select the window of frames to draw the random sample from? 

Thanks!

**Comment by gitttt-1234** [comment]
Closing this discussion, this feature is added in #2707  :)!
